helm/prometheus-rules/templates/alerting-rules/service-level.rules.yml

```diff
@@ -18,13 +18,13 @@ spec:
         slo_errors_per_request:ratio_rate1h{service!~"efk-.*|.*external-dns.*|kong-.*|.*nginx-.*"}
         and on (service)
         (
-        slo_errors_per_request:ratio_rate1h > on (service) group_left (cluster_type, cluster_id, class) slo_threshold_high
```

Comment:
@giantswarm/sig-monitoring What do you think about this solution? It gets rid of the error when I run it in the browser, but I don't know if it affects something else inadvertently. `slo_threshold_high` doesn't have any labels.

Reply:
I guess only @giantswarm/team-atlas can answer this question. I have no clue

Reply:
I'm actually not sure this changes anything as the rules are defined in a tenant cluster prometheus anyway. The class should stay though

Reply:
> I'm actually not sure this changes anything as the rules are defined in a tenant cluster prometheus anyway.

@QuentinBisson Could you elaborate on this?